hil-test/Cargo.toml

```diff
@@ -227,3 +231,6 @@ incremental      = false
 opt-level        = 3
 lto              = "fat"
 overflow-checks  = false
+
+[lints.rust]
+unexpected_cfgs = "allow"
```

Comment:
Why is this necessary?

Reply:
Because esp-meta emit's cfgs that rust doesn't know about in build.rs

Reply:
In that case I'd prefer unlinting this in build.rs and not crate-wide, I think.

Reply:
Sorry, I'm not being clear. It emits cfgs for the entire crate from build.rs. We have the same allow rule in esp-hal.

Reply:
Right, that makes it clearer, thanks